Dshop/apps/users/forms.py

```diff
@@ -0,0 +1,54 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
+from django.forms import ModelForm
+
+from .models import User, CustomUser
+
+
+class CustomUserForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ['username', 'password1', 'password2', 'email']
+
+    def __init__(self, *args, **kwargs):
+        super(CustomUserForm, self).__init__(*args, **kwargs)
+        self.fields['username'].error_messages = {
+            'required': 'custom required message'}
+        self.fields['password1'].error_messages = {
+            'required': 'custom required password'}
+        self.fields['email'].error_messages = {
+            'required': 'custom required email'}
+        self.fields['username'].widget.attrs.update({'class': 'form-control',
```

Comment:
This is pretty clever. 
One improvement could be going over the fields in loop and updating class. But: good.